### ISSUE TITLE
NPM-package is missing typescript definitions and es-modules

### ISSUE THREAD

**Issue by benbender** [prograsdk/react-use-search#16]
Hey, thanks for your work!

Sadly I noticed that the published npm-package (v0.2.3) is lacking typescript definitions and es-modules. Would be wonderful if you could rerelease the package with a fixed build!

````
❯ ll node_modules/react-use-search/dist/
total 32
-rw-r--r--  1 bb  staff    12K 17 Aug 15:19 index.js
````

**Comment by frederikspang** [comment]
Hi @benbender 

I've assigned my colleague Emil, and given him a Ping on our internal slack. 

Will you be able to submit a PR, changing the configuration to include the types?

Sincerely,
Frederik Spang
progras ApS

**Comment by emilbaekdahl** [comment]
@benbender This is fixed in the latest release (v0.2.5). Thanks for filing the issue!
